dist/css/yeldaWebchatInjector.css

```diff
@@ -81,8 +85,12 @@
     right: 20px;
     bottom: 20px;
     z-index: 2147483647;
-	}
-  
+  }
+
+  .yelda_container.inner{
```

Comment:
Why is this selector two times ? can you make one that works for both @media cases ? thanks